### PR TITLE
Fails with strict refs

### PR DESCRIPTION
**Pull request by pauloscustodio**

### CODE REVIEW

**Comment by karenetheridge** [comment]
If this wasn't visibly causing warnings before, why is this change needed?

File: lib/namespace/clean.pm [hunk 1]
```diff
@@ -37,6 +37,7 @@ use namespace::clean::_Util qw( DEBUGGER_NEEDS_CV_RENAME DEBUGGER_NEEDS_CV_PIVOT
 #
 
 my $RemoveSubs = sub {
+	no strict 'refs';
```
Comment (haarg) [inline lib/namespace/clean.pm]: This is using a tab for indentation, but should be using 4 spaces.

**Comment by haarg** [comment]
The released version is not broken, but it is broken in git. Not sure how that slipped through.

**Comment by haarg** [comment]
Thanks.

**Pull request merged**